features/torque2/server/rpms/config.pan

```diff
@@ -1,4 +1,16 @@
 unique template features/torque2/server/rpms/config;
 
-'/software/packages/{emi-torque-server}' = nlist();
-'/software/packages/{emi-torque-client}' = nlist();
+prefix '/software/packages';
+
+'{emi-version}' ?= dict();
+'{glite-yaim-torque-client}' ?= dict();
+'{glite-yaim-torque-server}' ?= dict();
+'{glite-yaim-torque-utils}' ?= dict();
+'{libtorque}' ?= dict();
+'{maui}' ?= dict();
+'{maui-client}' ?= dict();
+'{maui-server}' ?= dict();
+'{torque}' ?= dict();
+'{torque-client}' ?= dict();
+'{torque-mom}' ?= dict();
+'{torque-server}' ?= dict();
```

Comment:
This is ok, but for future compatibility this would be better as:

```pan
'/software/packages' = {
    pkg_repl('emi-version');
    pkg_repl('glite-yaim-torque-client');
    … etc …
};
```